compiler/qsc_eval/src/state/tests.rs

```diff
@@ -594,7 +594,7 @@ fn check_get_latex_for_decimal() {
     );
     assert_latex_for_decimal(
         &expect!([r#"
-            "1"
+            "1.0000"
```

Comment:
If the value is 1, seems we'd want to show `1`, not `1.0000`.

Reply:
Changed, I was kind of trying to match it with Python's output when printing state for consistency in state visualization.
![image](https://github.com/microsoft/qsharp/assets/40084144/5be5a185-1bc5-43e8-9bba-9c28c3f212e8)